common/src/main/java/com/revenuecat/purchases/common/diagnostics/DiagnosticsTracker.kt

```diff
@@ -37,6 +37,61 @@ class DiagnosticsTracker(
         )
     }
 
+    fun trackGoogleQuerySkuDetailsRequest(
+        skuType: String,
+        billingResponseCode: Int,
+        billingDebugMessage: String,
+        responseTimeMillis: Long
+    ) {
+        trackEvent(
+            DiagnosticsEntry.Event(
+                name = DiagnosticsEventName.GOOGLE_QUERY_SKU_DETAILS_REQUEST,
+                properties = mapOf(
+                    "sku_type_queried" to skuType,
+                    "billing_response_code" to billingResponseCode,
+                    "billing_debug_message" to billingDebugMessage,
+                    "response_time_millis" to responseTimeMillis
+                )
+            )
+        )
+    }
+
+    fun trackGoogleQueryPurchasesRequest(
+        skuType: String,
+        billingResponseCode: Int,
+        billingDebugMessage: String,
+        responseTimeMillis: Long
+    ) {
+        trackEvent(
+            DiagnosticsEntry.Event(
+                name = DiagnosticsEventName.GOOGLE_QUERY_PURCHASES_REQUEST,
+                properties = mapOf(
+                    "sku_type_queried" to skuType,
+                    "billing_response_code" to billingResponseCode,
+                    "billing_debug_message" to billingDebugMessage,
+                    "response_time_millis" to responseTimeMillis
+                )
+            )
+        )
+    }
+
+    fun trackGoogleQueryPurchaseHistoryRequest(
+        billingResponseCode: Int,
+        billingDebugMessage: String,
+        responseTimeMillis: Long
+    ) {
+        trackEvent(
+            DiagnosticsEntry.Event(
+                name = DiagnosticsEventName.GOOGLE_QUERY_PURCHASE_HISTORY_REQUEST,
+                properties = mapOf(
+                    "billing_response_code" to billingResponseCode,
+                    "billing_debug_message" to billingDebugMessage,
+                    "response_time_millis" to responseTimeMillis
```

Comment:
`"response_time_millis"` is repeated in all of these, but it seems like a common parameter.
Would it make sense to extract a constant?